x-pack/legacy/plugins/siem/public/pages/detection_engine/translations.ts

```diff
@@ -19,7 +19,7 @@ export const TOTAL_SIGNAL = i18n.translate('xpack.siem.detectionEngine.totalSign
 });
 
 export const SIGNAL = i18n.translate('xpack.siem.detectionEngine.signalTitle', {
-  defaultMessage: 'Signals (SIEM Detections)',
+  defaultMessage: 'Detected signals',
```

Comment:
I'm assuming this change was requested from UI/UX and product?

Just pointing it out in case this is a mistake. Otherwise I would put a bullet point about it changing in the main PR.

Reply:
This was a request from UI/UX. I added a note about it in the PR summary.